proxy/envoy/config_test.go

```diff
@@ -236,6 +236,7 @@ func testConfig(r *model.IstioRegistry, mesh *proxyconfig.ProxyMeshConfig, insta
 		Config:     r,
 		MeshConfig: mesh,
 		IPAddress:  instance,
+		UID:        fmt.Sprintf("uid://%s", instance),
```

Comment:
ah, good point. The mock stuff has no names but we it would clear the confusion to have something at least.